test/e2e/workflow/deployment_test.go

```diff
@@ -160,6 +161,50 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			CheckConfigCondition(ConditionAvailable, ConditionTrue, 10*time.Minute, 30*time.Second)
 		})
 	})
+
+	Context("testing range generation after kubeMacPool reinstalation ", func() {
+		var (
+			config                                           opv1alpha1.NetworkAddonsConfigSpec
+			oldRangeEnd, oldRangeStart, rangeEnd, rangeStart string
+		)
+
+		BeforeEach(func() {
+			config = opv1alpha1.NetworkAddonsConfigSpec{KubeMacPool: &opv1alpha1.KubeMacPool{}}
+			CreateConfig(config)
```

Comment:
this is racefull, if you wan't to prepare a configuration, you should wait until it is installed. if you are interested in testing the progress of the installation, CreateConfig should be part of It